tmscti/delivery_methods/tms_an22_function.sqf

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Please rename this file to something more descriptive, leaving out the tms_ prefix.